Project/Project/Pages/Index.cshtml

```diff
@@ -21,22 +38,25 @@
         </form>
     </div>
 </div>
-
 @if (Model.SearchCompleted)
 {
-    <table class="table">
-        <thead>
+    <table class="table" style="width:120%">
```

Comment:
Inline styles are not XHTML 1.0 Strict compliant.  For XHTML 1.0 Strict compliance, styles should not be in HTML markup, but should be in linked stylesheets.